roles/reproducer/tasks/prepare_networking.yml

```diff
@@ -152,6 +152,45 @@
       server=/{{ _domain }}/127.0.0.2
       server=/apps-crc.testing/127.0.0.2
       server=/crc.testing/127.0.0.2
+      server=/utility/127.0.0.2
+
+- name: Feed utility records
+  block:
+    - name: Ensure "utility" domain is local
+      become: true
+      notify: "Restart dnsmasq"
+      ansible.builtin.copy:
+        dest: "/etc/cifmw-dnsmasq.d/utility.conf"
+        content: |-
+          local=/utility/
+
+    # Using ctlplane gateway IP should ensure we're targeting an IP
+    # on the hypervisor, reachable by the VMs, ensuring it can discuss
+    # with the host (vbmc and sushy_emulator reaching libvirt, etc).
+    - name: Build utility records dataset
+      vars:
+        _target_net: "{{ cifmw_networking_env_definition.networks.ctlplane }}"
+        _controller_net: >-
+          {{
+            cifmw_networking_env_definition.instances['controller-0'].networks
+          }}
+        cifmw_dnsmasq_host_record:
+          - names:
+              - "ipmi.utility"
```

Comment:
ipmi.utility can't be on the ctlplane - no access from the ocp-bootstrap, it's plugged on ocpbm and ocppr only... dang.

Reply:
That is not a problem, we have routing enabled.

Reply:
nah, I'll just use the "public" net, I see we have it, so all good.